templates/shared_services/gitea/terraform/gitea-webapp.tf

```diff
@@ -34,7 +34,7 @@ resource "azurerm_app_service" "gitea" {
     GITEA_PASSWD   = "@Microsoft.KeyVault(SecretUri=${azurerm_key_vault_secret.gitea_password.id})"
     GITEA_EMAIL    = "giteaadmin@tre.com"
```

Comment:
I noticed earlier that username gitea_admin did not work (giteaadmin did work). Can we align the username and email?

Reply:
fixed